Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -120,10 +123,11 @@ public function apply(ResourceInterface $resource, QueryBuilder $queryBuilder)
                     $value = $this->getFilterValueFromUrl($value);
                 }
 
-                $valueParameter = QueryUtils::generateParameterName($field);
+                $valueParameter = QueryUtils::generateParameterName($this->isPropertyNested($property) ? $alias.'_'.$field : $field);
```

Comment:
Uhh, that's only a limitation of the mock. Perhaps we can improve that to produce sequential values (or something like that, not sure what `PHPMock` can do...)? I don't think it's a real concern that there will be collisions in the actual values from `uniqid`.